easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,163 @@
+    }),
+    ('contextlib2', '21.6.0', {
+        'checksums': ['ab1e2bfe1d01d968e1b7e8d9023bc51ef3509bba217bb730cee3827e1ee82869'],
+    }),
+    ('ml_collections', '0.1.1', {
+        'preinstallopts': "touch requirements.txt && touch requirements-test.txt && ",
+        'checksums': ['3fefcc72ec433aa1e5d32307a3e474bbb67f405be814ea52a2166bfc9dbe68cc'],
+    }),
+    (name, version, {
+        'patches': [
+            'AlphaFold-2.0.0_fix-packages.patch',
+            'AlphaFold-2.3.2_data-dep-paths-shebang-UniRef30_2023_02.patch',
+            'AlphaFold-2.0.0_n-cpu.patch',
+            'AlphaFold-2.0.1_setup_rm_tfcpu.patch',
+            'AlphaFold-2.3.2_use_openmm_8.0.0.patch',
+            'AlphaFold-2.3.2_BioPythonSCOPData.patch',
+        ],
+        'source_urls': ['https://github.com/deepmind/alphafold/archive/refs/tags/'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': [
+            {'AlphaFold-2.3.2.tar.gz': '4ea8005ba1b573fa1585e4c29b7d188c5cbfa59b4e4761c9f0c15c9db9584a8e'},
+            {'AlphaFold-2.0.0_fix-packages.patch': '826d2d1a5d6ac52c51a60ba210e1947d5631a1e2d76f8815305b5d23f74458db'},
+            {'AlphaFold-2.3.2_data-dep-paths-shebang-UniRef30_2023_02.patch':
+             '72a7b7cb8be4105f7bd32b404454afa465300d16b1ccc9933c9a70873f134f4e'},
+            {'AlphaFold-2.0.0_n-cpu.patch': 'dfda4dd5f9aba19fe2b6eb9a0ec583d12dcefdfee8ab8803fc57ad48d582db04'},
+            {'AlphaFold-2.0.1_setup_rm_tfcpu.patch':
+             '1a2e4e843bd9a4d15ee39e6c37cc63ba281311cc7a0a5610f0e43b52ef93faac'},
+            {'AlphaFold-2.3.2_use_openmm_8.0.0.patch':
+             'bbef940c0c959040aaf3984ec47777a229c164517b54616a2688d58fae636d84'},
+            {'AlphaFold-2.3.2_BioPythonSCOPData.patch':
+             '5aedd5a35da6d0a993ca19f4de807cf77ec36a5a52c97100110a0c5475cef1b8'},
+        ],
+    }),
+]
+
+local_pylibdir = '%(installdir)s/lib/python%(pyshortver)s/site-packages'
+local_link_scp = 'ln -s %%(installdir)s/stereo_chemical_props.txt %s/alphafold/common' % local_pylibdir
+
+postinstallcmds = [
+    'cp %(builddir)s/AlphaFold/alphafold-%(version)s/run_alphafold*.py %(installdir)s/bin',
+    'cp -rpP %(builddir)s/AlphaFold/alphafold-%(version)s/scripts %(installdir)s',
+    'cd %(installdir)s/bin && ln -s run_alphafold.py alphafold',
+    'chmod a+x %(installdir)s/bin/run_alphafold.py',
+    local_link_scp,
+]
+
+sanity_check_paths = {
+    'files': ['bin/alphafold', 'bin/pdbfixer', 'bin/run_alphafold.py', 'stereo_chemical_props.txt'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages', 'scripts'],
+}
+
+sanity_check_commands = [
+    "pdbfixer --help",
+    "python -m openmm.testInstallation",
+    "python -c 'import alphafold'",
+    "python %(installdir)s/bin/run_alphafold_test.py",
+    "alphafold --help 2>&1 | grep 'Full AlphaFold protein structure prediction script'",
```

Comment:
These two should probably be in the reverse order, make more sense to check if --help works before running a real test.